sypht/client.py

```diff
@@ -96,16 +98,23 @@ def _get_headers(self, **headers):
         })
         return headers
 
-    def upload(self, file, fieldset, tags=None, endpoint=None, workflow=None, options=None):
+    def upload(self, file, fieldset=None, fieldsets=None, tags=None, endpoint=None, workflow=None, options=None):
```

Comment:
I think we can simplify the API here to use a single, non-default parameter "fieldsets".

`def upload(self, file, fieldsets, ...)`

I'm not a huge fan of mixed type parameters, but I'm also not a fan of breaking changes to the API, so I would suggest doing something like:
```
if isinstance(fieldsets, str):
    fieldsets = [fieldsets]
```